lib/commons/aria/get-role-type.js

```diff
@@ -9,8 +9,13 @@ import lookupTable from './lookup-table';
  * @return {Mixed} String if a matching role and its type are found, otherwise `null`
  */
 function getRoleType(role) {
-	var r = lookupTable.role[role];
-	return (r && r.type) || null;
+	const roleDef = standards.ariaRoles[role];
+
+	if (!roleDef) {
+		return null;
+	}
+
+	return roleDef.type;
```

Comment:
Previously, this would return `null` if there wasn't a truthy `.type`. Do we need to maintain consistently here, or is the possibility of returning `undefined` here ok?

Reply:
Every role should have a type so it shouldn't be possible to return `undefined` or `null`. A role without a type should be seen as a bug.